hacs.json

```diff
@@ -1,10 +1,8 @@
-
 {
     "name": "vacuum-styj02ym",
     "country": ["FR", "PL"],
     "domains": ["vacuum"],
-    "homeassistant": "0.109.0",
-    "iot_class": "local_poll",
-    "zip_release": true,
-    "filename": "miio2.zip"
+    "iot_class": "local_polling",
```

Comment:
I think it should be `"Local Polling"` based on the documentation:
* https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things
* https://hacs.xyz/docs/publish/start/